.github/CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+# Contributing Guidelines
```

Comment:
Copy from other repo

Reply:
gonna keep so we know that we need contributing guidelines still or to update them